### PR TITLE
Dtspb 2447 ee cw create

### PR DESCRIPTION
**Pull request by itoulou**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPB-2447

### Change description ###
new ee iht fields on case worker create manual evebt

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/probate/controller/BusinessValidationController.java [hunk 1]
```diff
@@ -377,6 +377,12 @@
 
         CallbackResponse response;
 
+        response = eventValidationService.validateRequest(callbackRequest, allCaseworkerAmendValidationRules);
```
Comment (itoulou) [inline src/main/java/uk/gov/hmcts/probate/controller/BusinessValidationController.java]: Amend rule to CW amend to create

File: src/test/java/uk/gov/hmcts/probate/validator/IHTValidationRuleTest.java [hunk 2]
```diff
@@ -73,16 +74,34 @@ public void testValidateWithSuccessWhenIhtIsNull() {
...
+            .thenReturn(businessValidationError);
+
+        List<FieldErrorResponse> validationError = underTest.validate(ccdDataMock);
+
+        assertThat(validationError.isEmpty(), is(false));
+        verify(businessValidationMessageService, times(1))
+            .generateError(BUSINESS_ERROR, IHT_ESTATE_NET_GREATER_THAN_GROSS);
         assertTrue(validationError.contains(businessValidationError));
     }
 }
```
Comment (itoulou) [inline src/test/java/uk/gov/hmcts/probate/validator/IHTValidationRuleTest.java:119]: Worth adding a test if Gross and net values are equal for both probate and iht values

File: ccdImports/configFiles/CCD_Probate_Backoffice/CaseEventToFields.json [hunk 3]
```diff
@@ -211,7 +212,7 @@
...
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boAmendCaseDetails", "CaseFieldID": "bulkScanCaseReference", "PageFieldDisplayOrder": 7, "DisplayContext": "READONLY", "PageID": "boAmendCaseDetailsPage1", "PageDisplayOrder": 1, "ShowSummaryChangeOption": "n"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boAmendCaseDetails", "CaseFieldID": "dateOfDeathType", "PageFieldDisplayOrder": 1, "DisplayContext": "MANDATORY", "PageID": "boAmendCaseDetailsPage2", "PageDisplayOrder": 2, "PageShowCondition": "selectionList=\"deceasedIHT\"", "ShowSummaryChangeOption": "y"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boAmendCaseDetails", "CaseFieldID": "dateOfDeathType", "PageFieldDisplayOrder": 1, "DisplayContext": "MANDATORY", "PageID": "boAmendCaseDetailsPage2", "PageDisplayOrder": 2, "PageShowCondition": "selectionList=\"deceasedIHT\"", "CallBackURLMidEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/case/sols-default-iht-estate", "ShowSummaryChangeOption": "y"},
```
Comment (itoulou) [inline ccdImports/configFiles/CCD_Probate_Backoffice/CaseEventToFields.json:215]: Rename and refactor url call back - remove "sol" prefix

File: ccdImports/configFiles/CCD_Probate_Backoffice/CaseEventToFields.json [hunk 4]
```diff
@@ -2929,22 +3051,31 @@
...
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "applyforGrantPaperApplicationMan", "CaseFieldID": "appointExecNo", "PageFieldDisplayOrder": 126, "DisplayContext": "OPTIONAL", "PageID": "applyforGrantPaperApplicationManPage1", "PageDisplayOrder": 1, "FieldShowCondition": "caseType=\"gop\" AND englishWill=\"Yes\" AND appointExec=\"No\"", "ShowSummaryChangeOption": "y"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "applyforGrantPaperApplicationMan", "CaseFieldID": "domicilityCountry", "PageFieldDisplayOrder": 128, "DisplayContext": "MANDATORY", "PageID": "applyforGrantPaperApplicationManPage1", "PageLabel": "Foreign domicile", "PageDisplayOrder": 1, "FieldShowCondition": "deceasedDomicileInEngWales=\"No\"", "ShowSummaryChangeOption": "y"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "applyforGrantPaperApplicationMan", "CaseFieldID": "labelInheTax", "PageFieldDisplayOrder": 129, "DisplayContext": "READONLY", "PageID": "applyforGrantPaperApplicationManPage1", "PageLabel": "Foreign domicile", "PageDisplayOrder": 1, "ShowSummaryChangeOption": "y"},
```
Comment (itoulou) [inline ccdImports/configFiles/CCD_Probate_Backoffice/CaseEventToFields.json:2932]: Double check with business that this is only applied to Manual create event

Comment (itoulou) [reply ccdImports/configFiles/CCD_Probate_Backoffice/CaseEventToFields.json:2932]: And that the full case amend is also not affected

**Review by sanjayparekh-hmcts — APPROVED** [review]

**Pull request merged**